ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/ParsePayloadTest.java

```diff
@@ -153,7 +153,7 @@ public void testVersionInPayload() {
         + ",\"document_type\":\"main\"" + "},\"payload\":\"eyJ2ZXJzaW9uIjo0fQ==\"}";
 
     WithErrors.Result<PCollection<PubsubMessage>> result = pipeline.apply(Create.of(input))
-        .apply(InputFileFormat.json.decode()).output()
+        .apply(InputFileFormat.json.decode())
```

Comment:
Out of curiosity, what's the rule for putting `//` at the end of a line? Seems like it's sometimes there when `apply`ing multiple transforms and sometimes it isn't

Reply:
It's purely there to force google-java-format to allow a line break. There are lots of cases where it's not actually needed since the following line is already long enough that the formatter would have put it on its own line anyway. I personally tend to add `//` wherever I want a linebreak to be whether the formatter would remove it or not. There's definitely some personal preference involved.

Reply:
Generally, the longer the list of `apply`s, the more likely that I'll want to add `//` to force each `apply` to its own line to make it easier to skim the code and quickly understand the sequence of transforms.